syntax = "proto2";

package PluginInfoProto;

message PluginInfo
{

    message EnvironmentVariablesT
    {
        optional string name = 1;
        optional string value = 2;
    }

    repeated string policyAppIds = 1;
    repeated string statusAppIds = 2;
    optional string pluginName = 3;
    optional string xmlTranslatorPath = 4;
    optional string executableFullPath = 5;
    repeated string executableArguments = 6;
    repeated EnvironmentVariablesT environmentVariables = 7;
    optional string executableUserAndGroup = 8;
    optional int64  secondsToShutDown = 9  [default=2];
}
